src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,270 @@
+package duke;
+
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.Todo;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class Duke {
+    private static ArrayList<Task> tasks = new ArrayList<>();
+    private static Scanner in = new Scanner(System.in);
+    private static final String FILE_PATH = "duke.txt";
+
+    private static void printDuke() {
+        String logo = " ____        _\n"
+                + "|  _ \\ _   _| | _____\n"
+                + "| | | | | | | |/ / _ \\\n"
+                + "| |_| | |_| |   <  __/\n"
+                + "|____/ \\__,_|_|\\_\\___|\n";
+        System.out.println("Hello from\n" + logo + "\n");
+    }
+
+    private static String breakLine() {
+        return "\t--------------------------------------------------------------\n";
+    }
+
+    // Greet
+    private static void greet() {
+        System.out.println(breakLine()
+                + "\tHello! I'm Duke :)\n"
+                + "\tWhat can I do for you?\n"
+                + breakLine());
+    }
+
+    // read existing data
+    private static void getFileData() {
+        File file = new File(FILE_PATH);
+        System.out.println("\tLoading existing data . . .\n");
+        String input;
+        try {
+            if (!file.createNewFile()) {
+                Scanner fileInput = new Scanner(file);
+                while (fileInput.hasNext()) {
+                    input = fileInput.nextLine();
+                    String[] inputArgs = input.split(" \\| ");
+                    addFileDataToList(inputArgs);
+                }
+            }
+        } catch (IOException e) {
+            System.out.print("\t(!) IOException error: get existing file data.\n" + breakLine());
+        } catch (IllegalStateException e) {
+            System.out.print("\t(!) IllegalStateException: (!) Invalid file contents.\n" + breakLine());
+        }
+        System.out.print("\tLoading Done :)\n" + breakLine());
```

Comment:
Missing a finally clause to run final segment of code regardless of outcome of try catch statement